src/lib/components/infomessage/InfoMessageUtils.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { defaultTheme } from '../../style/theme';
+import React from 'react';
+import { useComputeBackgroundColor } from './InfoMessageUtils';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CoreUiThemeProvider } from '../coreuithemeprovider/CoreUiThemeProvider';
+
+describe('useComputeBackgroundColor', () => {
+  const SUT = jest.fn();
+
```

Comment:
As it is extracted you will need to add: 

afterEach(() => {
  SUT.clearMocks()
})